examples/sample-app/cleanup.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 echo "Killing openshift all-in-one server ..."
-killall openshift
+pkill --exact openshift
```

Comment:
@smarterclayton does this option exists on Mac? (I have bad feeling that you have just `-x`)